### PR TITLE
EDU-416 Use more custom icons

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-416

Please see ticket for screenshots of replaced icons.

### CODE REVIEW

File: src/components/icons/user-activities/item-edited.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (ahelmberger) [inline src/components/icons/user-activities/item-edited.js:1]: Seems like this is the only one that does not have a file name ending in `-icon.js`

File: test-app/src/bundles/page-header.js [hunk 2]
```diff
@@ -120,7 +118,7 @@ function PageHeader({ onUiLanguageClick }) {
             <Login />
           </div>
           <LinkPopover items={pageMenuItems} trigger="click" placement="bottomRight">
-            <Button className="PageHeader-headerButton" icon={<MenuOutlined />} />
+            <Button className="PageHeader-headerButton" icon={<MenuIcon />} type="link" />
```
Comment (ahelmberger) [inline test-app/src/bundles/page-header.js:121]: Link :+1:

**Comment by ahelmberger** [comment]
Looks nice :heart_eyes:

**Review by ahelmberger — APPROVED** [review]

File: src/components/icons/user-activities/item-edited.js (hunk 1 above)
Comment (adrianaluchian) [reply src/components/icons/user-activities/item-edited.js]: Good catch, thanks! Corrected

**Pull request merged**